docs/END-TO-END-TESTING.md

```diff
@@ -89,3 +99,6 @@ The client id for Plio backend application.
 
 #### `BACKEND_DEFAULT_OAUTH2_CLIENT_SECRET`
 The client secret for Plio backend application.
+
+#### `BACKEND_SECRET_KEY`
+This variable sets the secret key for the backend application so that it can be installed properly.
```

Comment:
for each backend env variable, let's add a link to backend's `ENV.md` file

Reply:
Done.